.agents/skills/create-repository/SKILL.md

```
@@ -0,0 +1,178 @@
+
+```bash
+./gradlew :data:assembleDebug -q
+```
+
+Hilt processes `@Binds` at compile time, so this proves the binding is well-formed (the impl is `@Inject`-able, return type matches the interface, no duplicate binding). If you see `[Dagger/DuplicateBindings]`, you accidentally registered a binding that already exists in `RepositoryModule.kt` — investigate the legacy module before retrying.
+
+# Output to the user
+
+Concise checklist after success:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)